CHANGES_NEXT_RELEASE

```diff
@@ -1,3 +1,4 @@
 - Add: -mqttTimeout (env var ORION_MQTT_TIMEOUT) for MQTT broker connection timeout
 - Fix: queue workers deadlock when MQTT broker was not responding to connection attemp
 - Fix: Ngsiv2-AttrsFormat header set to "normalized" in NGSI patching custom notification
+- Fix: Orion should return error and refuse to start when port is negative
```

Comment:
Just a small suggestion to include the related issue number, pls:

```suggestion
- Fix: Orion should return error and refuse to start when port is negative (3875)
```

Reply:
Fixed in [5cd0e52](https://github.com/telefonicaid/fiware-orion/pull/4287/commits/5cd0e52824c063e8b0d7638ea875ab2bf4e15341)